Services/Infrastructure/OJS.Services.Infrastructure/BackgroundJobs/Implementations/HangfireBackgroundJobsService.cs

```diff
@@ -1,14 +1,22 @@
 namespace OJS.Services.Infrastructure.BackgroundJobs.Implementations
 {
     using Hangfire;
+    using Hangfire.Common;
+    using Hangfire.States;
+    using OJS.Common.Enumerations;
     using System;
     using System.Linq.Expressions;
 
     public class HangfireBackgroundJobsService : IHangfireBackgroundJobsService
     {
         public object AddFireAndForgetJob<T>(
             Expression<Action<T>> methodCall)
-            => BackgroundJob.Enqueue(methodCall);
+        {
+            var client = new BackgroundJobClient();
+            var job = Job.FromExpression(methodCall);
+
+            return client.Create(job, new EnqueuedState(ApplicationName.Administration.ToString().ToLower()));
+        }
```

Comment:
Add queue as parameter. We are not sure that queue will always be Administration

Reply:
Fixed.